src/helpers/skins.ts

```diff
@@ -1,11 +1 @@
-const requireFile = require.context(
-  '@snapshot-labs/snapshot-spaces/skins/',
```

Comment:
I can't a way to do this with `import.meta.glob` because globbing from `node_modules` isn't supported somehow.